### PR TITLE
fix issue #30

### PR DESCRIPTION
**Pull request by fzzr-**
It was happening when & because my `.bash_profile` contains libs like a `echo "anything"`.

### CODE REVIEW

File: lib/linter-perl.coffee [hunk 1]
```diff
@@ -66,12 +66,13 @@ module.exports = class LinterPerl
               [lineNum-1, 0]
               [lineNum-1, buffer.lineLengthForRow(lineNum-1)]
             ]
-          results.push {
-            type: 'Error'
-            text: message
-            filePath
-            range
-          }
+          if range != null and (message != null and message.length)
```
Comment (s-shin) [inline lib/linter-perl.coffee]: Maybe this can be made simple like `if range and message?.length`.

**Comment by s-shin** [comment]
> my `.bash_profile` contains libs like a `echo "anything"`

Oh, I see.

I think this PR will be acceptable, but could you fix the part of my comment and combine the commits to one?

**Comment by fzzr-** [comment]
@s-shin I did it. I hate to change history :)

**Comment by s-shin** [comment]
Thanks.

> I hate to change history :)

I also agree in some case :D

**Pull request merged**